File: Ejercicios Resueltos/Lenguaje C/Ejercicios Basicos/Ejercicio-27/ejercicio27.c
// Ejercicio 27
// Hacer un programa que calcule el salario de un empleado, si se descuenta el 20% de su
// salario bruto.


#include <stdio.h>

void main(){

    float sueldoBruto, sueldoNeto;

    printf("Ingrese sueldo Bruto: ");
    scanf("%f",&sueldoBruto);

    sueldoNeto=sueldoBruto-(sueldoBruto*0.2);

    printf("Sueldo Bruto: $%.2f\n",sueldoBruto);
    printf("Sueldo Neto: $%.2f\n",sueldoNeto);

    printf("\nPresione ENTER para continuar...");
    getchar();
    getchar();

}

File: Ejercicios Resueltos/Lenguaje Python/Ejercicios Basicos/Ejercicio-27/ejercicio27.py
# Ejercicio 27
# Hacer un programa que calcule el salario de un empleado, que descuenta el 20% de su
# salario bruto.

print("EJERCICIO 27\n\n\n")
sueldo = float(input("Ingrese el sueldo: "))
sueldo = sueldo - sueldo * 0.2
sueldo = round(sueldo,2)
print(f'\nEl sueldo a pagar con 20% de descuento es de ${sueldo}')

input("\n\nPresione ENTER para continuar...")